climpred/smoothing.py

```diff
@@ -132,145 +134,103 @@ def check_lon_lat_present(da):
+        )
+    if not ('time' in tsmooth_kws or 'lead' in tsmooth_kws):
         raise ValueError(
-            'smooth_kws doesnt contain a time dimension \
+            'tsmooth_kws doesnt contain a time dimension \
             (either "lead" or "time").',
-            smooth_kws,
+            tsmooth_kws,
         )
-    smooth = list(smooth_kws.values())[0]
-    dim = list(smooth_kws.keys())[0]
+    smooth = list(tsmooth_kws.values())[0]
+    dim = list(tsmooth_kws.keys())[0]
     # fix to smooth either lead or time depending
     time_dims = ['time', 'lead']
     if dim not in ds.dims:
         time_dims.remove(dim)
         dim = time_dims[0]
-        smooth_kws = {dim: smooth}
+        tsmooth_kws = {dim: smooth}
     # aggreate based on how
-    ds_smoothed = getattr(ds.rolling(smooth_kws, center=False), how)()
+    ds_smoothed = getattr(ds.rolling(tsmooth_kws, center=False), how)()
     # remove first all-nans
     ds_smoothed = ds_smoothed.isel({dim: slice(smooth - 1, None)})
     ds_smoothed[dim] = ds.isel({dim: slice(None, -smooth + 1)})[dim]
-    if rename_dim:
-        ds_smoothed = _reset_temporal_axis(ds_smoothed, smooth_kws=smooth_kws, dim=dim)
     return ds_smoothed
 
 
-@is_xarray(0)
-def _reset_temporal_axis(ds_smoothed, smooth_kws=None, dim=None):
-    """Reduce and reset temporal axis. See temporal_smoothing(). Might be
+def _reset_temporal_axis(ds_smoothed, tsmooth_kws, dim='lead', set_lead_center=True):
+    """Reduce and reset temporal axis. See temporal_smoothing(). Should be
     used after calculation of skill to maintain readable labels for skill
     computation.
 
     Args:
         ds_smoothed (xarray object): Smoothed dataset.
-        smooth_kws (dict): Keywords smoothing is performed over.
-            Default is {'time': 4.
-        dim (str): Dimension smoothing is performed over ('time' or 'lead').
+        tsmooth_kws (dict): Keywords smoothing is performed over.
+        dim (str): Dimension smoothing is performed over. Defaults to 'lead'.
+        set_center (bool): Whether to set new coord `{dim}_center`.
+            Defaults to True.
 
     Returns:
         Smoothed Dataset with updated labels for smoothed temporal dimension.
     """
-    if smooth_kws is None:
-        smooth_kws = {'time': 4}
-    if not ('time' in smooth_kws or 'lead' in smooth_kws):
-        raise ValueError('smooth_kws doesnt contain a time dimension.', smooth_kws)
-    smooth = list(smooth_kws.values())[0]
-    dim = list(smooth_kws.keys())[0]
-    try:
-        # TODO: This assumes that smoothing is only done in years. Is this fair?
-        composite_values = ds_smoothed[dim].to_index().year
-    except AttributeError:
-        composite_values = ds_smoothed[dim].values
-    new_time = [f'{t}-{t + smooth - 1}' for t in composite_values]
-    ds_smoothed[dim] = new_time
+    # bugfix: actually tsmooth_kws should only dict
+    if tsmooth_kws is None or callable(tsmooth_kws):
+        return ds_smoothed
```

Comment:
this is a fix I needed to add because sometimes `self._temporally_smoothed` just changed in `pytest climpred/tests/test_hindcastEnsemble_class.py::test_verify`. dont understand why.